std/conv.d

```diff
@@ -260,11 +261,12 @@ template to(T)
 
 /**
  * When converting strings _to numeric types, note that the D hexadecimal and binary
- * literals are not handled. Neither the prefixes that indicate the base, nor the
+ * literals are $(B not) handled. Neither the prefixes that indicate the base, nor the
  * horizontal bar used _to separate groups of digits are recognized. This also
- * applies to the suffixes that indicate the type.
+ * applies _to the suffixes that indicate the type.
  *
- * _To work around this, you can specify a radix for conversions involving numbers.
+ * To work around this, you can specify a radix for conversions involving numbers:
+ * $(D_CODE T $(DDOC_PSYMBOL _to)$(DDOC_TEMPLATE_PARAM_LIST (S))(S value, uint radix);)
```

Comment:
Looks a bit weird on the docs:

![image](https://user-images.githubusercontent.com/4370550/35538838-845dc01a-054f-11e8-8d81-842116b281c1.png)

and again no precedent of manually specifying the overload in the plain-text docs.